src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -78,6 +78,8 @@
     )
 
     var (
+        Done = errors.New("iterator done")
```

Comment:
The reason why we use io.EOF is
- it's already builtin and known by programmers probably
- gRPC streaming is using the same error

Also -- defining a Done value per client seems redundant to me. If doing so, it would be better to define it in gax-golang, and use it here.